hudi-common/src/test/java/org/apache/hudi/common/util/TestTablePathUtils.java

```diff
@@ -87,19 +92,22 @@ static void setup() throws IOException {
 
   @Test
   void getTablePathFromTablePath() throws IOException {
+    setup();
```

Comment:
can we make the setup() BeforeEach instead of BeforeAll ?

Reply:
In one of the setup, I need to pass the the file format.